site/kubernetes/kube-addon.md

```diff
@@ -186,12 +186,42 @@ definition](https://v1-7.docs.kubernetes.io/docs/api-reference/v1.7/#networkpoli
 
 ## <a name="troubleshooting"></a> Troubleshooting
 
+The first thing to check is whether the Weave Net software is up and
+running. The `kubectl apply` command you use to install only
+_requests_ that it be downloaded and started; if anything goes wrong
+at startup the details will only be visible in the logs of the container(s).
+
```

Comment:
The first thing to check is whether Weave Net is up and
running. The `kubectl apply` command you used to install it only
_requests_ that it be downloaded and started; if anything goes wrong
at startup, those details will only be visible in the logs of the container(s).

Suggest: include a link on how to get the logs of a container?

Reply:
The `kubectl logs` command 20 lines down is how.

Reply:
Probably a good idea to link to it with an anchor tag.

Reply:
I missed that. Looks good!